### ISSUE TITLE
dateutil is not a valid package

### ISSUE THREAD

**Issue by mbhall88** [esteinig/sketchy#18]
https://github.com/esteinig/sketchy/blob/66a2004223bcd72481f5cda875d39341f15b3f2a/requirements.txt#L8

`dateutil` is not an actual package. I believe you might mean [`python-dateutil`](https://pypi.org/project/python-dateutil/)?  

It seems like you added it at the same time you added the anaconda package to your conda yaml file. The confusion seems to stem from the naming of the anaconda package. See [this StackOverflow question](https://stackoverflow.com/questions/37776256/conda-says-conflict-over-installing-dateutil-in-python-3-5-1) and answer for further details.

As it stands, trying to install in a local virtual environment from the `requirements.txt` file gives the following error

```
$ pipenv install -r requirements.txt
Creating a virtualenv for this project…
Pipfile: /home/michael/Projects/sketchy/Pipfile
Using /home/linuxbrew/.linuxbrew/bin/python3.7 (3.7.3) to create virtualenv…
⠹ Creating virtual environment...Using base prefix '/home/linuxbrew/.linuxbrew/bin/../Cellar/python/3.7.3'
New python executable in /home/michael/.local/share/virtualenvs/sketchy-wT9ylmut/bin/python3.7
Also creating executable in /home/michael/.local/share/virtualenvs/sketchy-wT9ylmut/bin/python
Installing setuptools, pip, wheel...
done.
Running virtualenv with interpreter /home/linuxbrew/.linuxbrew/bin/python3.7

✔ Successfully created virtual environment! 
Virtualenv location: /home/michael/.local/share/virtualenvs/sketchy-wT9ylmut
Requirements file provided! Importing into Pipfile…
Pipfile.lock not found, creating…
Locking [dev-packages] dependencies…
Locking [packages] dependencies…
✘ Locking Failed! 
[pipenv.exceptions.ResolutionFailure]:   File "/home/linuxbrew/.linuxbrew/Cellar/pipenv/2018.11.26_2/libexec/lib/python3.7/site-packages/pipenv/resolver.py", line 69, in resolve
[pipenv.exceptions.ResolutionFailure]:       req_dir=requirements_dir
[pipenv.exceptions.ResolutionFailure]:   File "/home/linuxbrew/.linuxbrew/Cellar/pipenv/2018.11.26_2/libexec/lib/python3.7/site-packages/pipenv/utils.py", line 726, in resolve_deps
[pipenv.exceptions.ResolutionFailure]:       req_dir=req_dir,
[pipenv.exceptions.ResolutionFailure]:   File "/home/linuxbrew/.linuxbrew/Cellar/pipenv/2018.11.26_2/libexec/lib/python3.7/site-packages/pipenv/utils.py", line 480, in actually_resolve_deps
[pipenv.exceptions.ResolutionFailure]:       resolved_tree = resolver.resolve()
[pipenv.exceptions.ResolutionFailure]:   File "/home/linuxbrew/.linuxbrew/Cellar/pipenv/2018.11.26_2/libexec/lib/python3.7/site-packages/pipenv/utils.py", line 395, in resolve
[pipenv.exceptions.ResolutionFailure]:       raise ResolutionFailure(message=str(e))
[pipenv.exceptions.ResolutionFailure]:       pipenv.exceptions.ResolutionFailure: ERROR: ERROR: Could not find a version that matches dateutil
[pipenv.exceptions.ResolutionFailure]:       No versions found
[pipenv.exceptions.ResolutionFailure]: Warning: Your dependencies could not be resolved. You likely have a mismatch in your sub-dependencies.
  First try clearing your dependency cache with $ pipenv lock --clear, then try the original command again.
 Alternatively, you can use $ pipenv install --skip-lock to bypass this mechanism, then run $ pipenv graph to inspect the situation.
  Hint: try $ pipenv lock --pre if it is a pre-release dependency.
ERROR: ERROR: Could not find a version that matches dateutil
No versions found
Was https://pypi.org/simple reachable?
[pipenv.exceptions.ResolutionFailure]:       req_dir=requirements_dir
[pipenv.exceptions.ResolutionFailure]:   File "/home/linuxbrew/.linuxbrew/Cellar/pipenv/2018.11.26_2/libexec/lib/python3.7/site-packages/pipenv/utils.py", line 726, in resolve_deps
[pipenv.exceptions.ResolutionFailure]:       req_dir=req_dir,
[pipenv.exceptions.ResolutionFailure]:   File "/home/linuxbrew/.linuxbrew/Cellar/pipenv/2018.11.26_2/libexec/lib/python3.7/site-packages/pipenv/utils.py", line 480, in actually_resolve_deps
[pipenv.exceptions.ResolutionFailure]:       resolved_tree = resolver.resolve()
[pipenv.exceptions.ResolutionFailure]:   File "/home/linuxbrew/.linuxbrew/Cellar/pipenv/2018.11.26_2/libexec/lib/python3.7/site-packages/pipenv/utils.py", line 395, in resolve
[pipenv.exceptions.ResolutionFailure]:       raise ResolutionFailure(message=str(e))
[pipenv.exceptions.ResolutionFailure]:       pipenv.exceptions.ResolutionFailure: ERROR: ERROR: Could not find a version that matches dateutil
[pipenv.exceptions.ResolutionFailure]:       No versions found
[pipenv.exceptions.ResolutionFailure]: Warning: Your dependencies could not be resolved. You likely have a mismatch in your sub-dependencies.
  First try clearing your dependency cache with $ pipenv lock --clear, then try the original command again.
 Alternatively, you can use $ pipenv install --skip-lock to bypass this mechanism, then run $ pipenv graph to inspect the situation.
  Hint: try $ pipenv lock --pre if it is a pre-release dependency.
ERROR: ERROR: Could not find a version that matches dateutil
No versions found
Was https://pypi.org/simple reachable?
```

And then a search on `pypi`
```sh
$ pip3 search dateutil
dateutil-br (0.0.7)           - Implementation to Pt-br of python lib python-dateutil
pyfortified-dateutil (0.1.1)  - Extension to Python `datetime` functionality.
py-dateutil (2.2)             - Extensions to the standard Python datetime module
python-dateutil (2.8.0)       - Extensions to the standard Python datetime module
  INSTALLED: 2.8.0 (latest)
dateutilsmit (0.0.1)          - MIT Wrapper for dateutils
relativetimebuilder (2.0.0)   - A library for using the dateutil relativedeltas for calendar precision with aniso8601
dates (0.0.1)                 - A small tool based on python-dateutil that just prints dates
dtxg (2.0.1)                  - Extensions to the dateutil module to support time resolution in multiple languages
rrule34 (1.0.0)               - Python dateutil rrule tools.Noticeably natural language formatting of recurences.
bdateutil (0.1)               - Adds business day logic and improved data type flexibility to python-dateutil.
```

**Comment by esteinig** [comment]
Yep sorry my bad, should be `python-dateutil` 

Fix incoming once I land in Darwin.

**Comment by mbhall88** [comment]
I already have a fix in #19 that I am just waiting for you to review and merge.
